### PR TITLE
feat(google-mail): action script to send an email using google-mail integration

### PR DESCRIPTION
**Pull request by Pyloris**
## Describe your changes
- Modified nango.yaml file of `/integration-templates/google-mail/nango.yaml` and added 2 models.
- Added an Action script `/integration-templates/google-mail/actions/send-email.ts` for sending an email.
- 
## Issue ticket number and link
#2241
ISSUE TICKET: [NAN-1075](https://linear.app/nango/issue/NAN-1075/google-mail-integration-does-not-support-sending-email)

### CODE REVIEW

File: CONTRIBUTING.md
Comment (bodinsamuel) [inline CONTRIBUTING.md:1]: This file was discussed in a previous PR, can you remove this change please? ☺️

File: integration-templates/google-mail/actions/send-email.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            endpoint: '/gmail/v1/users/me/messages/send',
+            data: {
+                raw: base64EncodedEmail
+            }
+        });
+
+        return mapEmail(sentEmailResponse.data);
+    } catch (error: any) {
+        throw new nango.ActionError({
+            message: 'Failed to fetch fields in the gmail-send action script.',
```
Comment (bodinsamuel) [inline integration-templates/google-mail/actions/send-email.ts]:
```suggestion
            message: 'Failed to send email in the gmail-send action script.',
```

**Review by bodinsamuel — APPROVED** [review]
Looks good to me 🚀

File: CONTRIBUTING.md
Comment (Pyloris) [reply CONTRIBUTING.md:1]: I have reverted the changes

**Review by khaliqgant — APPROVED** [review]
Looks good minus the incorrect ActionError message

**Comment by Pyloris** [comment]
@khaliqgant @bodinsamuel issues have been fixed.

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**